client/package.json

```diff
@@ -144,7 +144,7 @@
     "prom-client": "^12.0.0",
     "proxy-agent": "^5.0.0",
     "querystringify": "^2.1.1",
-    "re2": "^1.16.0",
+    "re2": "^1.20.9",
```

Comment:
If anyone is wondering why we are leaving re2 in the client side code, it is oddly used to statically generate the regex url parsing by the `url-regex-safe` package (weird for a client side package, I know, but it works).